app/Http/Controllers/Api/ReactionController.php

```diff
@@ -24,6 +25,22 @@ public function __construct()
     }
```

Comment:
@sbsmith86 @katiecrane do we want this endpoint to be available to unauthorized users? If so, we might want to add middleware back into the `routes` file and maybe make a separate section for all `GET` endpoints (like `GET` `/activity`)?